### DISCUSSION TITLE
hide the snow

### DISCUSSION THREAD

**Discussion by hairullana** [dbeaver/dbeaver#22287] [General]
anyone knows how to hide the snow?
that's annoying and make my laptop lags when opened dbeaver at the first time
![image](https://github.com/dbeaver/dbeaver/assets/56705867/b4656ec7-3a7e-4c91-8908-18b4b54a6dae)

**Comment by LonwoLonwo** [comment]
Hello @hairullana 

You can turn it off in Preferences -> User Interface -> Miscellaneous.

![2023-12-29 11_03_03-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/6701a4e2-6248-4303-b2af-f99f1c99a9b2)
